datasets/coached_conv_pref/README.md

```diff
@@ -15,8 +15,12 @@ source_datasets:
 - original
 task_categories:
 - other
+- sequence-modeling
+- structure-prediction
 task_ids:
-- other-other-Coached Conversation Preference
+- other-other-Conversational Recommendation
+- dialogue-modeling
+- parsing
```

Comment:
Not sure why there's `parsing` here ?

Reply:
That's from me since there's a structured representation with all of the utterances :)

Reply:
Ok I see !

Reply:
Should we keep the definition of the `parsing` tag as extracting the syntactic structure (constituency parsing), and use another tag for this type of utterances parsing ?

Reply:
not sure yet. We should probably revisit that when we've tagged most of our existing dataset!

Reply:
Right. Let's keep it this way for now then :)